pkg/agent/endpoints/sdsv2/handler.go

```diff
@@ -204,6 +213,10 @@ func (h *Handler) DeltaSecrets(discovery_v2.SecretDiscoveryService_DeltaSecretsS
 }
```

Comment:
Shall we also return the same `Unavailable` code for the `DeltaSecrets` method when the SDS v2 API is not enabled to be consistent across all the methods in the SDS v2 API?